taxonomies/nutrients.txt

```diff
@@ -5896,10 +5896,26 @@ unit:en: %
 zz:fruits-vegetables-nuts-estimate-from-ingredients
 en:Fruits\, vegetables\, nuts and rapeseed\, walnut and olive oils (estimate from ingredients list analysis)
 xx:Fruits\, vegetables\, nuts and rapeseed\, walnut and olive oils (estimate from ingredients list analysis)
+ar:الفواكه \ ، الخضار \ ، المكسرات والبذور \ ، زيوت الجوز والزيتون (تقدير من تحليل قائمة المكونات)
```

Comment:
will this kind of comma escape work ? 

It seems you could instead use their own sign (and avoid the need for escaping): https://blogs.transparent.com/arabic/punctuation-in-arabic/